go/enclave/crosschain/message_bus_manager.go

```diff
@@ -156,6 +156,8 @@ func (m *MessageBusManager) RetrieveInboundMessages(fromBlock *common.L1Block, t
 		b = p
 	}
 
+	m.logger.Trace(fmt.Sprintf("Extracted deposit logs %d ->%d: %v.", fromBlock.NumberU64(), toBlock.NumberU64(), len(messages)), log.CmpKey, log.CrossChainCmp)
```

Comment:
out of curiosity. Why not %d for the len(messages)?

Reply:
It wasn't len initially so I've missed it when changing haha